ports/hpx/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "hpx",
-  "version-string": "1.7.0",
+  "version-string": "1.7.1",
```

```suggestion
  "version-semver": "1.7.1",
```

This field should be served by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#version-semver).